package.json

```
@@ -18,10 +18,12 @@
   "files": [
     "index.js"
```

The typings would need to be included in the files release to go out with a release.

```suggestion
    "index.js",
    "types/index.d.ts"
```